### ISSUE TITLE
Code too large for try statement

### ISSUE THREAD

**Issue by bzp99** [OpenJML/OpenJML#805]
I am trying to compile some relatively complex code with the latest release, but started running into

```
error: code too large for try statement
```

The compiler shows these for every line of my source file. The thing is, I have removed all try-catch blocks from the reported file and I still get the error. Compilation succeeds with pure OpenJDK.

Could the cause be that some of the methods are quite long (~200–250 LOC)?

**Comment by bzp99** [comment]
Commenting out the long methods fix the error. I suppose the fact that I have moved from public class fields to private fields, getters/setters, and builders introduced a lot more code suddenly and that lead to the error. This is still an issue though, because I don't know if segmenting the code more will help.

**Comment by bzp99** [comment]
I have broken up my code further, which seems to have fixed the issue.

That said, I still believe this is a bug and this behaviour should at least be documented.

**Comment by davidcok** [comment]
Indeed, OPenJML introduces its own try-catch-finally blocks, and those can contain a l;arge amount of code.
The workaround is to keep methods small.
However this issue does point out the need to do more to make the rac-translation of a ,java file more modular itself.
There is no example here to make into a test case. Leaving this issue open as a reminder of the design problem.

**Comment by davidcok** [comment]
A while ago, a change was implemented that encapsulates the runtime evaluation of invariants in synthesized methods. This reduces the code size, which should help with this problem.
